src/file-loader.test.ts

```diff
@@ -0,0 +1,256 @@
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
this was added to test a bad configuration, I know we must trust on TS, but on the file we have many `any`s already, but was used for the last 2 tests, it kinda was just to add more branch coverage